src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -265,6 +266,14 @@ const formConfig = {
           uiSchema: uploadPersonalPtsdDocuments.uiSchema,
           schema: uploadPersonalPtsdDocuments.schema,
         },
+        physicalHealthChanges: {
+          title: 'Additional Remarks - Physical Health Changes',
+          path: 'new-disabilities/ptsd-781a-physical-changes',
+          depends: formData =>
+            hasNewPtsdDisability(formData) && needsToEnter781a(formData),
```

Comment:
These conditionals will cause the user to always see this page even if they choose to upload a 781a. Should they only see this page if they choose to answer questions? If so we need a depends to check for that. something like
```
isAnsweringPtsdForm = formData =>
  _.get('view:uploadPtsdChoice', formData, '') === 'answerQuestions';
```

Reply:
Fixed!